client/odie/query/index.ts

```diff
@@ -42,28 +43,97 @@ function odieWpcomSendSupportMessage( message: Message, path: string ) {
 	} );
 }
 
-// It will post a new message using the current chat_id
+// Internal helper function to generate a uuid
```

Comment:
Should this use the `uuid` lib included in Calypso?

Reply:
I  removed it looking forward to generating a package with the least dependencies as possible. 

I wouldn't like to have a few extra imports due to minimal code. InsteadI want to implement that minimal code.